docs/the-new-architecture/landing-page.md

```diff
@@ -1,32 +1,72 @@
 
-<NewArchitectureWarning/>
+The team is now working to make this the default experience for the React Native open source ecosystem. The term _New Architecture_ refers to both the new framework architecture and the work to bring it to open source.
 
-Starting from version 0.68, React Native provides the New Architecture, which offers developers new capabilities for building highly performant and responsive apps. Visit [Why a New Architecture](why) to learn more about what drove the decision to re-architect and the benefits it provides.
+The New Architecture has been available for experimental opt-in as of [React Native 0.68](/blog/2022/03/30/version-068#opting-in-to-the-new-architecture) with continued improvements. The remaining work of the rollout is to ensure that the thousands of React Native libraries that developers depend on work out-of-the-box or have a straightforward migration path.
 
-To achieve these benefits, we had to rethink how Native Modules and Native Components work. This led us to develop the [Pillars of the New Architecture](pillars):
+## Why a New Architecture?
 
-- [The New Native Module System - Turbo Modules](pillars-turbomodules), a framework to support efficient and flexible integration with native code
-- [The New Native Renderer - Fabric](pillars-fabric-components), which offer improved capabilities, cross-platform consistency, and performance in rendering
-- [The Codegen](pillars-codegen), which generates boilerplate C++ required by the New Architecture via static typing in JavaScript
+Primarily, the New Architecture is an investment of the future of React Native to write polished app experiences. The New Architecture unlocks capabilities and improvements that were not possible in the legacy architecture.
 
-## Get Started with the New Architecture
+### Synchronous execution
 
-### For App Developers
+Prior, calling a native module method required passing a [callback](https://reactnative.dev/docs/next/native-modules-ios#callbacks) function to receive any data from native. Synchronous execution simplifies your native interfacing logic to provide a better experience for your app.
 
-To **create a new app** using the New Architecture, head over to [Creating a New Architecture App](use-app-template), which will get you up and running in a few quick steps with the new app template.
+In particular, synchronous rendering is a huge experience win for the New Architecture. Synchronous layout of your React views eliminates a class of bugs related to layout jumps and race conditions related to invalid layout data. Long-standing bugs with list positions and virtualization can be fixed holistically.
 
-To **migrate an existing app** to the New Architecture, follow [Adopting the New Architecture](../new-architecture-intro).
+### Support for React 18 concurrent features
```

Comment:
Can we include code for the example? It's not clear to me which feature this demo is using. It makes it sound as if this is an improvement/breakage that you will get out of the box (untrue) vs the reality that you opt-in with a feature.

Reply:
I think that is the case though here, where using the new renderer is throwing away intermediate work. Let me try and emphasize that more here